src/cls/rbd/cls_rbd_client.h

```diff
@@ -1,17 +1,17 @@
-// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*- 
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
 // vim: ts=8 sw=2 smarttab
 
 #ifndef CEPH_LIBRBD_CLS_RBD_CLIENT_H
 #define CEPH_LIBRBD_CLS_RBD_CLIENT_H
 
 #include "cls/lock/cls_lock_types.h"
 #include "cls/rbd/cls_rbd_types.h"
-#include "common/bit_vector.hpp"
```

Comment:
@dillaman Now `test_cls_rbd` fails with the error "‘ceph::BitVector<2> ref_bit_vector’ has incomplete type" due to the header is not included.

Reply:
Updated